extension/src/util/generateRandomPipeName.ts

```diff
@@ -0,0 +1,30 @@
+import { randomBytes } from "crypto";
+import * as os from "os";
+import * as path from "path";
+import * as fs from "fs";
+
+const XDG_RUNTIME_DIR = process.env["XDG_RUNTIME_DIR"];
+const safeIpcPathLengths: Map<NodeJS.Platform, number> = new Map([
+    ["linux", 107],
+    ["darwin", 103],
+]);
+
+export function generateRandomPipeName(): string {
```

Comment:
It should be possible to directly import this util: https://github.com/redhat-developer/vscode-java/blob/38c8b582b40db9644696924f899c73a3251563f4/src/javaServerStarter.ts#L9

Reply:
Currently use local implementation and mark as TODO, should be replace after upgrade `"vscode-languageclient": "7.0.0"` to higher version